### PR TITLE
Setup promex

### PR DESCRIPTION
**Pull request by manusajith**
### Changes

Setup promex for gathering application insights

Below you'll find a checklist. For each item on the list, check one option and delete the other.

### Tests
- [x] This PR does not require tests

### Changelog
- [ ] Entry has been added to changelog
- [ ] This PR does not make a user-facing change

### Documentation
- [ ] [Docs](https://github.com/plausible/docs) have been updated
- [ ] This change does not need a documentation update

### Dark mode
- [x] This PR does not change the UI

### CODE REVIEW

File: lib/plausible/prom_ex.ex [hunk 1]
```diff
@@ -0,0 +1,102 @@
+defmodule Plausible.PromEx do
+  @moduledoc """
```
Comment (ukutaht) [inline lib/plausible/prom_ex.ex:2]: This reads like generic Promex documentation which mostly is implemented in this PR. Is there any value in leaving this @moduledoc in?

File: lib/plausible/prom_ex.ex [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  @impl true
+  def dashboards do
+    [
+      # PromEx built in Grafana dashboards
+      {:prom_ex, "application.json"},
+      {:prom_ex, "beam.json"},
+      {:prom_ex, "phoenix.json"},
+      {:prom_ex, "ecto.json"},
+      {:prom_ex, "oban.json"}
+      # {:prom_ex, "phoenix_live_view.json"},
```
Comment (ukutaht) [inline lib/plausible/prom_ex.ex:94]: Not a fan of these comments. If we want to add stuff later we can just look up the docs for Promex, no?

**Review by ukutaht — COMMENTED** [review]
Looks good. I would just cut down on some of the generic comments in the Promex module. Seems like leftover from copying an example file?

File: lib/plausible/prom_ex.ex (hunk 1 above)
Comment (manusajith) [reply lib/plausible/prom_ex.ex]: The file was automatically generated using the generator `mix prom_ex.gen.config ... `

Just cleaned it up.

**Pull request merged**